components/secondary_button/README.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Secondary buttons allow users to carry out a supporting task
```

Comment:
This description should match the description language below. Here you say task, but below you say action. Action is probably the better term